internal/changelog/entry.go

```diff
@@ -32,7 +30,7 @@ func EntryFromFragment(f fragment.File) Entry {
 		Summary:     f.Fragment.Summary,
 		Description: f.Fragment.Description,
 		Component:   f.Fragment.Component,
-		LinkedPR:    f.Fragment.Pr,
+		LinkedPR:    []int{f.Fragment.Pr},
```

Comment:
What does happen when `f.Fragment.Pr` is `0` empty? I expect this line initialize `LinkedPR` with `[]int{0}`, which is not what we want.

To avoid this is probably needed to check for `f.Fragment.Pr` value and initialize `LinkedPR` to empty slice or value (if `> 0`).